src/ReactWrapper.jsx

```diff
@@ -722,11 +722,19 @@ class ReactWrapper {
    * @param {*} initialValue - the initial value
    * @returns {*}
    */
-  reduce(fn, initialValue) {
-    return this.getNodes().reduce(
-      (accum, n, i) => fn.call(this, accum, this.wrap(n), i),
-      initialValue,
-    );
+  reduce(fn) {
```

Comment:
I can't do that because the native reduce/reduceRight distinguish between `undefined` and "absent" - or rather, id still have to check `arguments.length` and ignore `initialValue` when `arguments.length` is < 2